charts/nx-cloud/templates/nx-cloud-remix-frontend-deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if not .Values.useLegacyFrontend }}
```

Comment:
this is only if they use the latest Helm chart but the old images?

Reply:
yes `useLegacyFrontend` has been included as a last resort, but it is not documented and carries no long-term guarantee that it will be present in future releases.